### PR TITLE
[SIRo] Collision Measure

### PR DESCRIPTION
**Pull request by ASzot**
New collision measure.

### CODE REVIEW

**Review by akshararai — APPROVED** [review]
looks good, thanks for adding this!

File: habitat-baselines/habitat_baselines/rl/multi_agent/measures.py [hunk 1]
```diff
@@ -42,6 +42,30 @@ def update_metric(self, *args, task, **kwargs):
         self._metric = found_contact
 
 
+@registry.register_measure
+class NumCollisions(Measure):
```
Comment (rpartsey) [inline habitat-baselines/habitat_baselines/rl/multi_agent/measures.py:46]: Hey @ASzot, shouldn't this measure be named NumAgentsCollide?

File: habitat-lab/habitat/config/default_structured_configs.py [hunk 2]
```diff
@@ -1129,6 +1129,11 @@ class DidAgentsCollideConfig(MeasurementConfig):
     type: str = "DidAgentsCollide"
 
 
+@dataclass
+class NumAgentsCollideConfig(MeasurementConfig):
+    type: str = "NumAgentsCollide"
```
Comment (rpartsey) [inline habitat-lab/habitat/config/default_structured_configs.py:1134]: Or type here set to NumCollisions?

File: habitat-baselines/habitat_baselines/rl/multi_agent/measures.py (hunk 1 above)
Comment (akshararai) [reply habitat-baselines/habitat_baselines/rl/multi_agent/measures.py:46]: Will fix and make a new PR, thanks for catching @rpartsey !

**Pull request merged**